reference-architecture/gce-ansible/playbooks/openshift-setup.yaml

```diff
@@ -6,23 +6,18 @@
     openshift_debug_level: "{{ debug_level }}"
     openshift_node_debug_level: "{{ node_debug_level | default(debug_level, true) }}"
     osm_controller_args:
-      cloud-provider:
-      - "gce"
       cloud-config:
-      - "/etc/origin/gce.ini"
+      - "/etc/gce/gce.conf"
     osm_api_server_args:
-      cloud-provider:
-      - "gce"
       cloud-config:
-      - "/etc/origin/gce.ini"
+      - "/etc/gce/gce.conf"
     openshift_node_kubelet_args:
-      cloud-provider:
-      - "gce"
       cloud-config:
-      - "/etc/origin/gce.ini"
+      - "/etc/gce/gce.conf"
```

Comment:
you shouldn't need to set osm_controller_args, osm_api_server_args, or openshift_node_kubelet_args. you'll probably need to munge the created cloud config file put down, but we should really add the support needed directly to openshift-ansible.